File: Data Structures & Algorithms/remove-element/submission-0.java
class Solution {
    public int removeElement(int[] nums, int val) {
        List<Integer> tempList = new ArrayList<>();
        for (int num : nums) {
            if (num != val) {
                tempList.add(num);
            }
        }
        for (int i = 0; i < tempList.size(); i++) {
            nums[i] = tempList.get(i);
            // This allows you to replace the elements within the Integer array with 
            // The elements that we want from the temporary list that we created
        }
        return tempList.size();
        // Finally, we just return the size
    }
}

File: Data Structures & Algorithms/find-all-numbers-disappeared-in-an-array/submission-0.py
class Solution:
    def findDisappearedNumbers(self, nums: List[int]) -> List[int]:
        n = len(nums)
        temp_set = set(range(1, n+1))
        for num in nums:
            if num in temp_set:
                temp_set.discard(num)
        return list(temp_set) 

        